gitlab/mixins.py

```diff
@@ -223,6 +223,18 @@ def get_update_attrs(self):
         """
         return getattr(self, '_update_attrs', (tuple(), tuple()))
 
+    def get_update_method(self):
+        """Return the HTTP method to use.
+
+        Returns:
+            object: http_put (default) or http_post
+        """
+        if getattr(self, '_update_post', False):
```

Comment:
I'm not sure about the attribute name, it doesn't feel explicit enough. Maybe `_update_with_post` or `_update_uses_post` could be a little better?

Reply:
Changed to _update_uses_post for increased clarity.